official/cv/segment-anything/README.md

```diff
@@ -0,0 +1,258 @@
+```
+
+the fine-tuned model will be saved at the work_root specified in `configs/sa1b_point_finetune.yaml`. For a fast single image inference, please run,
+
+```shell
+python point_inference.py --checkpoint=your/path/to/ckpt
+```
+
+Below is an experimental result batch-prompted with 5 points and the model is trained at scale `vit_b`. The checkpoint can be downloaded [here](https://download-mindspore.osinfra.cn/toolkits/mindone/sam/sam_vitb_point_finetune_flare-898ae8f6.ckpt). 
+<div align="center">
+    <img alt="img.png" src="https://github.com/user-attachments/assets/c452ffe1-215e-47e4-b4cc-fb9baedfc07b" width="600"/>
+</div>
+
+Explore more interesting applications such as iterative positive and negative points prompting described in the following Demo Chapter.
+
+### Finetune with text-prompt
+*Note again that text-to-mask finetune is exploratory and not robust, and the official pytorch code is not release yet.*
+
+
+The training procedure described in the official SAM paper is quite interesting that does not require new text annotation. Specifically, for each manually collected mask with area larger than 1002 we extract the CLIP image embedding. Then, during training, we prompt SAM
+with the extracted CLIP image embeddings as text prompt input. At inference time we run text through CLIP’s text encoder and then give the resulting text embedding as a prompt to SAM
+
+The key that make the training procedure work is that CLIP’s image embeddings are trained to align with its text embeddings.
+
+This repository provides an implementation of text-to-mask finetune referring to the model structure and training procedure described in the official SAM paper and introduces a stronger multimodal encoder BLIP2 in addition to CLIP.
+
+A machine with **64G ascend memory** is required for text-prompt finetune.
+
+First download  SA-1B dataset and put it under `${project_root}/datasets/sa-1b`.
+
+for standalone finetune of SA-1B dataset with BLIP2 (CLIP is similar), please run:
+```shell
+python train.py -c configs/sa1b_text_finetune_blip2.yaml
+```
+the BLIP2 checkpoint and bert vocabulary.txt will be automatically downloaded at `./checkpoint_download/`
+
+for distributed finetune, please run:
+```shell
+msrun --worker_num=8 --local_worker_num=8 train.py -c configs/sa1b_text_finetune_blip2.yaml
+```
+the fine-tuned model will be saved at the work_root specified in `configs/sa1b_text_finetune.yaml`. For a fast single image inference, please run,
+
+```shell
+python text_inference.py --checkpoint=your/path/to/ckpt --text-prompt your_prompt
+```
+
+### Performance
+
+Experiments are tested on ascend910* with mindspore 2.3.1 graph mode.
+
+| model name | cards | batch size | graph compile | recompute | sink | jit level | s/step | img/s |
+|:----------:| :---: | :--------: | :-----------: | :-------: | :---: | :------: | :----: | :---: |
+| sam-vit-b finetune on COCO2017 | 1 | 16 | 3~4 mins | OFF | ON | O2 | 2.59 | 6.18 |
```

Comment:
better add a `dataset` column instead of writing the dataset name in `model_name` column